test/integration/oauth_basicauth_test.go

```diff
@@ -32,7 +33,12 @@ const (
 )
 
 var (
-	basicAuthCerts = map[string][]byte{
+	remotePrefix       = "remote"
```

Comment:
is there a way to avoid lots of package-level vars? these get shared across all integration tests